Korean translate

johndoe0x opened the pull request:
- modeling bitcoin contracts by timed automata
- paxos made simple
- SCP ietf draft

번역본 및 이미지 첨부합니다.

johndoe0x committed:
SCP IETF draft initial commit

johndoe0x committed:
SCP IETF draft initial commit

johndoe0x committed:
PAXOS MADE SIMPLE commit.

johndoe0x committed:
PAXOS MADE SIMPLE commit.

johndoe0x committed:
modeling bitcoin contracts by timed automata commit

johndoe0x committed:
modeling bitcoin contracts by timed automata commit

johndoe0x committed:
add images on modeling bitcoin contracts by Timed Automata, add translate phrase top of the docs

johndoe0x committed:
add images on modeling bitcoin contracts by Timed Automata, add translate phrase top of the docs

johndoe0x committed:
remove useless md files

johndoe0x committed:
Merge branch 'korean_translate' of https://github.com/bosnet/papers into korean_translate

johndoe0x committed:
remove files

sconeman reviewed (changes requested):
디렉토리와 파일 명을 기존에 있는  `The-Stellar-Consensus-Protocol`에 맞추는게 낫지 않을까요? 예를 들어,

`ModelingBitcoinContractsByTimedAutomata/modeling_bitcoin_contracts_by_Timed_Automata.md`
->  `Modeling-Bitcoin-Contracts-By-Timed-Automata/README.md`

이런 식으로요.

johndoe0x commented:
@astonshnam 디렉토리 이름은 그렇게 해도 될건데 
논문들인데 README라고 해야할까요?

johndoe0x committed:
change dirs name

sconeman commented:
@cmcm2222 그래야 마크다운으로 작성한 의미가 있지 않을까요? 브라우저로 읽을 수 있게 하려고 README.md을 작성하시는 거 아닌가요?

johndoe0x commented:
@astonshnam ??
브라우저로 읽는것과 README 파일 이름을 지정하는게 기능상 차이가 있냐고 물어보시는 건가요?? 
파일 이름이 다르다고 뭔가 기능상 차이가 있을거 같진 않은데요?

johndoe0x commented:
README 파일은 대부분 안내를 위한것이라고 알고 있어서 파일이름을 README로 하는것보다는 
그냥 논문 이름을 쓰는게 더 맞다고 생각했어요.
확장자가 어차피 마크다운파일인데 ㅎㅎ.

sconeman commented:
@cmcm2222 아 제가 잘못 알고 있었네요.. 파일명이 README가 아니어도 브라우저에서 잘 볼 수 있군요. README.md가 아니면 소스코드 형식으로 보이는줄 알았어요..

sconeman commented:
@cmcm2222 그러면 스텔라 논문의 README.md의 이름을 바꿔야 할까요?

sconeman commented:
@cmcm2222 그리고 image 이름 형식도 통일했으면 좋겠어요. SCP논문에서는 image01, image02... 이렇게 되어있네요.

johndoe0x commented:
@astonshnam 사실 그부분은 한번 의논 해봐야 될거라고 생각했었어요.
내용이 바뀌는게 아니니 일단 이번 PR 통해서 정한 다음에 수정해도 되지 않을까요?
@leejames00 님이 PR을 add readme 라고 올리신게 있어서 이미지 파일이나 번역 md 파일을 명기할때 규칙정도는 간단히 정하는게 좋아보여요

johndoe0x commented:
@astonshnam 넵 수정해서 다시 커밋 올릴게요 :)

sconeman commented:
@cmcm2222 README.md가 파일의 안내를 위한 것이라 논문의 이름을 쓰는게 더 맞다는 의견에는 동의합니다. 그런데 그렇게 하려면 마크다운 파일 명 규칙도 정해야겠네요. 그 부분은 다른 PR에서 논의해보아요.

johndoe0x committed:
unifying image file names

johndoe0x commented:
@astonshnam 이미지 파일 이름 형식과 디렉토리 형식은 통일해서 커밋 올렸습니다. :) 
확인 부탁드려요

sconeman committed:
Update image files name in modeling_bitcoin_contracts....md

modeling_bitcoin_contracts_by_Timed_Automata.md에 image1, image2...로 되어있는 것을 image01, image02...로 수정했어요.

johndoe0x commented:
👍 @astonshnam

sconeman reviewed (approved):
LGTM!! 내일 마크다운 파일 이름에 대해 논의해보아요.

johndoe0x commented:
@leejames00 @spikeekips @astonshnam 번역된 자료들의 이름을 README.md 는 안내문의 형식이라 
기존 번역된 논문에는 다른 형식으로 업로드 해야 될거 같아요. 
혹시 의견있으신지?

johndoe0x commented:
@astonshnam @leejames00 @spikeekips @soonkuk 
번역된 자료 이름들은 
예를 들어 Paxos_Made_Simple.md , Modeling_Bitcoin_Contracts_By_Timed_Automata.md 
이런식으로 full 제목을 적는게 어떨까 싶은데요 
혹시 다른 의견 있으신지??

leejames00 commented:
언더바 말고 -는 어떤가요? 
Paxos-Made-Simple.md

sconeman commented:
저도 @leejames00 님의 방식을 더 선호해요.

johndoe0x commented:
@astonshnam @leejames00 네 그렇게 수정하도록 할게요:)

johndoe0x committed:
change file name

johndoe0x committed:
change file name

johndoe0x committed:
Merge branch 'korean_translate' of https://github.com/bosnet/papers into korean_translate

johndoe0x committed:
remove duplicate md file

johndoe0x committed:
amend image file uri

johndoe0x commented:
@astonshnam @leejames00 수정 완료 했어요 
확인 부탁드립니다. 
마지막 확인하신 분이 merge 해주시면 더 감사드립니다 ㅎㅎ

leejames00 commented:
SCP paper도 이름 바꿔주셨네요 👍  감사합ㄴ다

sconeman commented:
@cmcm2222 Modeling-Bitcoin-Contracts_By_Timed_Automata.md -> Modeling-Bitcoin-Contracts-By-Timed-Automata.md
부탁드려요.

johndoe0x committed:
change file name

johndoe0x committed:
change file name

johndoe0x commented:
@astonshnam 👍  수정되었습니다. 머지 부탁드려요

sconeman commented:
Modeling-Bitcoin-Contracts_By-Timed-Automata.md 언더바가 아직 하나 더 남아있어요ㅎㅎ

johndoe0x committed:
change missed file name

sconeman commented:
확인했습니다. merge 들어갑니다~

johndoe0x commented:
@astonshnam 넵 finally 이제사 된거 같아요 ㅎㅎ 
어떠세요??
